tests/integration/Core/Framework/Plugin/KernelPluginLoader/ComposerPluginLoaderTest.php

```diff
@@ -92,12 +92,6 @@ public function testLoadsPlugins(): void
     {
         $this->loadComposerLoadedPluginFixture();
 
-        /**
-         * @deprecated tag:v6.7.0 - double check if we can increase composer constraint and remove this
-         * @see https://github.com/composer/composer/issues/12235
-         */
-        static::markTestSkipped('This test is not compatible with Composer 2.8.4');
```

Comment:
^2.8.5 should be OK, right? https://github.com/shopware/shopware/blob/trunk/composer.json#L83

Reply:
yep, we was just waiting for a fix and it was merged in 2.8.4